pkg/httputil/httputil.go

```diff
@@ -42,14 +42,14 @@ func GetBodyOK(httpClient *http.Client, apiURL string) ([]byte, error) {
 		return nil, err
 	}
 	defer DeferClose(res.Body)
-	if res.StatusCode >= 400 {
-		errMsg := fmt.Errorf("Error response %v URL %s", res.StatusCode, apiURL)
-		return nil, errMsg
-	}
 	body, err := ioutil.ReadAll(res.Body)
 	if err != nil {
 		return nil, err
 	}
+	if res.StatusCode >= 400 {
```

Comment:
why move these lines here?

Reply:
> why move these lines here?

Cause I need to add the response body into error Info even when the status code of response is larger than 400.
Currently if the code > 400, the response body wouldn't be read.